### PR TITLE
Document Sensu Core 1.3.0

### PR DESCRIPTION
**Pull request by portertech**
Document Sensu Core 1.3.0 for stable release.

### CODE REVIEW

File: content/sensu-core/1.3/_index.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: "Sensu Core"
+date: 2017-07-21T13:06:31-07:00
+weight: -100
+menu: "sensu-core-1.3"
+version: "1.3"
+product: "Sensu Core"
+tags: ["sensu", "core", "sensu core", "1.1", "index"]
```
Comment (cwjohnston) [inline content/sensu-core/1.3/_index.md]: I'm not clear on how these tags are actually used, but tag for `1.1` should be `1.3`

Comment (portertech) [reply content/sensu-core/1.3/_index.md]: interesting, so this was also missed for 1.1 -> 1.2

Comment (asachs01) [reply content/sensu-core/1.3/_index.md]: 👍 to that. We'll also have to open an issue and replace this in other versions :/

File: config.toml [hunk 2]
```diff
@@ -50,9 +50,10 @@ ignoreFiles = [ "sensu-doc-template.md" ]
         name = "Sensu Core"
         description = "The open source extensible monitoring framework."
         weight = 1
-        latest = "1.2"
```
Comment (cwjohnston) [inline config.toml:53]: Also need to update static.json to ensure `/sensu-core/latest/` is redirected to the latest version

**Review by cwjohnston — CHANGES_REQUESTED** [review]

File: config.toml (hunk 2 above)
Comment (portertech) [reply config.toml:53]: Done 👍

File: content/sensu-core/1.3/api/_index.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+title: "API"
+product: "Sensu Core"
+version: "1.3"
+weight: -1
+menu: 
+  sensu-core-1.3:
+    parent: api
+---
+
+{{< directoryListing "content/sensu-core/1.0/api" >}}
```
Comment (cwjohnston) [inline content/sensu-core/1.3/api/_index.md]: 1.0 -> 1.3

**Review by cwjohnston — CHANGES_REQUESTED** [review]

**Review by cwjohnston — APPROVED** [review]
I just found that we have a number of broken links in the OS X platform page, but I'm okay with shipping this change now and addressing those later. 👍

**Comment by portertech** [comment]
@cwjohnston thanks!

**Pull request merged**